src/drive/reducers/view.js

```diff
@@ -91,7 +91,7 @@ const fileCount = (state = null, action) => {
     case DESTROY_FILES_SUCCESS:
       return state - action.ids.length
     case RESTORE_FILES_SUCCESS:
-      return state + action.ids.length
+      return state - action.ids.length
```

Comment:
This change fixes an issue when restoring a file: the LoadMore button was displayed just after the restore, and it went in a loop of trying to load more files...

Reply:
This is hard to understand how `DESTROY_FILES_SUCCESS` and `RESTORE_FILES_SUCCESS` modify the state the same way.

BTW, I guess this change has nothing to do with the `Alerter` of `cozy/cozy-ui`?

Reply:
That's right, it looks weird, but apart from adding a comment, I don't see how I can make it clearer...

And yes, I stumbled upon that bug during testing of the alert fixes, and didn't feel like creating a separate PR for a single char change.

Reply:
Can we add a comment then? I don't understand how this bugfix work, some at least a hint in the code would be good :)